components/brave_rewards/core/wallet_provider/wallet_provider.h

```diff
@@ -39,6 +39,8 @@ class WalletProvider {
 
   virtual base::TimeDelta GetDelay() const;
 
+  virtual void AssignWalletLinks(mojom::ExternalWallet& external_wallet) = 0;
```

Comment:
Should we have this as a non-`virtual` function, and have `GetAccountUrl()`, and `GetActivityUrl()` as `virtual` member functions of `WalletProvider` instead?

Reply:
I can see arguments both ways here. I think for now though we should opt for the simpler API (a single `AssignWalletLinks` method).